src/app.constants.ts

```diff
@@ -132,6 +132,11 @@ export enum LOCALE {
   CY = "cy",
 }
 
+export const LANGUAGE_CODES: Record<string, string> = {
```

Comment:
I think we could make use of an enum or an interface here to make it less generic perhaps?
`export interface LanguageCodes {
  en: 'en-GB',
  cy: 'cy-CY'
}

export const LANGUAGE_CODES: LanguageCodes = {
  en: "en-GB",
  cy: "cy-CY",
}`